### PR TITLE
Rr 441 education and training tab short route answers

### PR DESCRIPTION
**Pull request by chrimesdev**
## Description 

Education and training tab short route answers for "Training and eduction interests in prison"

This is just for the UI, which currently is commented out, the next PR will include the correct data

https://dsdmoj.atlassian.net/browse/RR-441

### CODE REVIEW

File: server/views/pages/overview/partials/educationAndTrainingTab/_trainingAndEducationInterestsInPrison.njk [hunk 1]
```diff
@@ -13,8 +13,32 @@ asked at the Induction.
...
+  
+          <dl class="govuk-summary-list">
+            <div class="govuk-summary-list__row">
+              <dt class="govuk-summary-list__key">
+                Training and eduction interests
+              </dt>
+              <dd class="govuk-summary-list__value">
+                <ul class="govuk-list">
+                  {# TODO: Replace with the correct data value for training and education interests #}
+                  {% for education in educationAndTraining.data.shortQuestionSetAnswers.additionalTraining %}
```
Comment (chrimesdev) [inline server/views/pages/overview/partials/educationAndTrainingTab/_trainingAndEducationInterestsInPrison.njk:25]: This is the wrong data but just a placeholder, will use the correct data in the next PR.

This view is commented out so not currently displayed.

File: server/views/pages/overview/partials/educationAndTrainingTab/educationAndTrainingTabContents.njk [hunk 2]
```diff
@@ -16,7 +16,7 @@ The 'Training and eduction in prison' summary card is only shown if the short qu
   </section>
 {% else %}
   <section data-qa="qualifications-and-education-history-short-question-set">
-    {% include './_qualificationsAndEducationHistory_inductionShortQuestionSet.njk' %}
+    {% include './_inductionShortQuestionSet.njk' %}
```
Comment (nathanrussell-moj-digital) [inline server/views/pages/overview/partials/educationAndTrainingTab/educationAndTrainingTabContents.njk:19]: Can we rename this back please? I think the full name is better as it matches the corresponding file for the long journey, and it describes that it contains the Qualifications And Education History content (as opposed any other content for long vs. short question set)

Comment (chrimesdev) [reply server/views/pages/overview/partials/educationAndTrainingTab/educationAndTrainingTabContents.njk]: Fixed

**Review by nathanrussell-moj-digital — APPROVED** [review]
LGTM 👍

**Pull request merged**